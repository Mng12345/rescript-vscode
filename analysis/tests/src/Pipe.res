type m = {name: string}

module M1 = {
  type t = m

  let handle = (m: t) => {
    assert false
  }
}

module M2 = {
  type t = m

  let handle = (m: t) => {
    assert false
  }
}

let m: m = {
  name: "zm",
}

let m1: M1.t = {
  name: "zm",
}

let m2: M2.t = m1

// m1->
//     ^com
