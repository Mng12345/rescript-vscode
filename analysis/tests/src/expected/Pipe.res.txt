Complete src/Pipe.res 28:7
posCursor:[28:7] posNoWhite:[28:6] Found expr:[28:3->0:-1]
Completable: Cpath Value[m1]->
Package opens Pervasives.JsxModules.place holder
Resolved opens 1 pervasives
ContextPath Value[m1]->
ContextPath Value[m1]
Path m1
CPPipe env:Pipe
CPPipe type path:M1.t
CPPipe pathFromEnv:M1 found:true
Path M1.
[{
    "label": "M1.handle",
    "kind": 12,
    "tags": [],
    "detail": "t => 'a",
    "documentation": null
  }]

